src/GraphQL/update-user.js

```diff
@@ -0,0 +1,57 @@
+
+import { gql } from '@apollo/client'
+// import GET_USER from './queries'
+
+export default gql `
+  mutation ($id: ID!, $description: String!) {
```

Comment:
I think it'd be great to confirm with BE that this is the only expected mutation is the about me/description.

Reply:
agree